database/mariadb/mariadb.def

```diff
@@ -3,7 +3,7 @@ From: mariadb:10.3.9
 
 %post
 # <YOUR_USERNAME> is the user who will be executing the container,
-# just run: `whoami` and thats your username.
+# just run: `whoami` and that's your username.
 # eg. sed -ie "s/^#user.*/user = westleyk/" /etc/mysql/my.cnf
```

Comment:
Can we export an environment variable called `YOUR_USERNAME` with the username right before running `sed -ie "s/^#user.*/user = <YOUR_USERNAME>/" /etc/mysql/my.cnf` ?

Reply:
Good idea! :+1: